src/main/java/frc/util/Sensors/absoluteEncoder/AbsoluteEncoder.java

```diff
@@ -0,0 +1,29 @@
+package frc.util.Sensors.absoluteEncoder;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.wpilibj.DutyCycleEncoder;
+
+public class AbsoluteEncoder {
+    private DutyCycleEncoder encoder;
+    private double offset;
+    private int inversion;
+
+    public AbsoluteEncoder(int id, double offset, boolean inversion) {
+        encoder = new DutyCycleEncoder(id);
+        this.offset = offset;
+
+        if (inversion) {
+            this.inversion = -1;
+        } else {
+            this.inversion = 1;
+        }
+    }
+
+    public double getPosition() {
+        return encoder.get() * 360 * inversion - offset;
+    }
+
+    public double getNormalizedPosition() {
+        return MathUtil.inputModulus(getPosition(), -180, 180);
+    }
+}
```

Comment:
I don't think you'll need this file after you remove the encoder from the subsystem